### PR TITLE
Upgrading Helm and Kubernetes.

### PR DESCRIPTION
**Pull request by Tang8330**
Upgrading the following dependencies:
* Kubernetes
* Helm
* aws-cli (from v1 to v2), breaking changes should not impact us.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-ARG AWSCLI_VERSION=1.31.10
+ARG AWSCLI_VERSION=2.13.25-r0
 ARG HELM_PLUGINS_TO_INSTALL="https://github.com/databus23/helm-diff"
 
 #gcloud path
 ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
 
 #install packages
-RUN apk add --update --upgrade --no-cache jq bash curl git gettext libintl py-pip
+RUN apk add --update --upgrade --no-cache jq bash curl git gettext libintl py-pip aws-cli=${AWSCLI_VERSION}
```
Comment (nathan-artie) [inline Dockerfile:18]: It might be nice to standardize on either using `curl` or `wget` for our dockerfiles, we currently use both.

**Review by nathan-artie — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (Tang8330) [reply Dockerfile:18]: Yeah, I'll leave an issue with the main repo since this is a fork.

Comment (nathan-artie) [reply Dockerfile:18]: Ahh I hadn't picked up that, it was a also a chore that I've meaning to do for some of our own dockerfiles.

Comment (nathan-artie) [reply Dockerfile:18]: Oh you probably don't need to install `py-pip` then if you're not using it to install `awscli`.

Comment (Tang8330) [reply Dockerfile:18]: `py-pip` was already a dependency

Comment (Tang8330) [reply Dockerfile:18]: I emailed the author (who made me a contributor to the repo), but the CI checks are broken

**Pull request merged**